OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -78,9 +78,9 @@ int main() {
     }
     // Print out the DataTable to console.
```

Comment:
The comment wasn't accurate to begin with either. I think the logic here was to ensure the table is still empty following L52–77. I'll improve the comment and tests here.